### PR TITLE
Add flowbox target

### PR DESCRIPTION
**Pull request by DusKing1**
This was made for RSgoBlackbox community.
Only have:
- a flash chip,
- a beeper,
- an external button,
- a BMI270 gyro

and no motor output.
image
image

### CODE REVIEW

File: configs/default/NEUT-FLOWBOX.config [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+resource SPI_MOSI 3 B05
+resource FLASH_CS 1 A15
+resource GYRO_EXTI 1 A01
+resource GYRO_CS 1 A04
+resource PWM 1 B10
+
+# timer
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer A15 AF1
```
Comment (mikeller) [inline configs/default/NEUT-FLOWBOX.config:28]: What's the timers / DMAs used for in addition to the beeper?
Also, the PWM pin will need a timer to work.

**Comment by mikeller** [comment]
Interesting idea. :grinning: 

Happy to support this for now, but obviously this is outside of the range of hardware that Betaflight is targeting. So if future changes to Betaflight mean that this won't work with a future version of Betaflight there probably won't be any attempts to make this work again. But then I guess since this isn't really dependent on flight performance improvements / new flight related features, there is no need for this hardware to always run the latest version of Betaflight.

**Comment by DusKing1** [comment]
Yes, this make sense, but only a few features was involved in this hardware, it's quite hard to say in the future it will be outdated.

File: configs/default/NEUT-FLOWBOX.config (hunk 1 above)
Comment (DusKing1) [reply configs/default/NEUT-FLOWBOX.config:28]: For serial receivers. Thanks for your hint, will update.

**Comment by rhys100** [comment]
Hey @DusKing1 where does one buy these chips?

**Comment by DusKing1** [comment]
@rhys100 currently this hardware didn't get massive manufactured due to the chip lack. You can stay tuned to the RSgoBlackbox community in discord, or AliExpress.

**Pull request merged**